actions/pdf/support.py

```diff
@@ -30,3 +33,14 @@ def _get_pdf_content(filename: str):
     if pdf.length == 0:
         _read_text_from_pdf(filename, pdf)
     return pdf
+
+
+def _access_file(filename: str):
```

Comment:
If we need this in more than one of our actions packs we should look at adding this to the chat.get_file or adding another helper function in the sema4ai-actions library.